### PR TITLE
Update Ivan's PPP

### PR DESCRIPTION
**Pull request by hiivan**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 [[homepage](https://www.ivan-ang.com)]
 [[github](https://github.com/hiivan)]
-[[portfolio](team/ivan.md)]
+[[portfolio](team/hiivan)]
```
Comment (kailashgautham) [inline docs/AboutUs.md:41]: Should be hiivan.md!

**Review by kailashgautham — APPROVED** [review]
Your PPP looks great! Minor change to fix regd. the file name

File: docs/AboutUs.md (hunk 1 above)
Comment (hiivan) [reply docs/AboutUs.md]: Thanks for the catch! Will update.

**Review by g-tejas — APPROVED** [review]
LGTM!

**Pull request merged**